src/main/java/ttt/Game.java

```diff
@@ -0,0 +1,66 @@
+package ttt;
+
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.io.Reader;
+import java.io.Writer;
+
+import static ttt.PlayerSymbol.O;
+import static ttt.PlayerSymbol.X;
+
+public class Game {
+    private static final int PLAYER_ONE_INDEX = 0;
+    private static final int PLAYER_TWO_INDEX = 1;
+    private final Board board;
+    private Prompt gamePrompt;
+    private Player[] players;
+
+    public static void main(String... args) {
+        Reader commandLineReader = new InputStreamReader(System.in);
+        Writer commandLineWriter = new OutputStreamWriter(System.out);
+        Prompt prompt = new CommandPrompt(commandLineReader, commandLineWriter);
+
+        HumanPlayer player1 = new HumanPlayer(prompt, X);
+        HumanPlayer player2 = new HumanPlayer(prompt, O);
+
+        Game game = new Game(new Board(), prompt, player1, player2);
+        game.play();
+    }
+
+    public Game(Board board, Prompt gamePrompt, Player player1, Player player2) {
+        this.board = board;
+        this.gamePrompt = gamePrompt;
+        this.players = new Player[]{player1, player2};
+    }
+
+    public void play() {
+        int index = PLAYER_ONE_INDEX;
+
+        while (board.hasFreeSpace()) {
```

Comment:
Perhaps there's a better way to implement the control flow here. Can you find a way to remove the need for the break statement?